openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -284,3 +291,40 @@ def test_responses_to_scoped_requests(self, scope_statuses):
         self.assertEqual(response.status_code, scope_statuses.read_status)
         response = self.post_with_bearer_token('/oauth2-with-scope-test/', token=self.access_token.token)
         self.assertEqual(response.status_code, scope_statuses.write_status)
+
+
+@ddt.ddt
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class TestJWTAuthentication(mixins.JwtMixin, TestCase):
```

Comment:
i think this test should be renamed to reflect that it's only testing the feature flag on JWT auth, not the authentication itself (which is provided by a third party package)